### PR TITLE
Improve Swift topic

### PR DESCRIPTION
**Pull request by akashivskyy**
- [x] I followed the contributing guidelines: [CONTRIBUTING.md](https://github.com/github/explore/blob/master/CONTRIBUTING.md)

I am:
  - [x] Suggesting edits to an existing Topic page
  - [ ] Curating a new Topic page

**EDITING AN EXISTING TOPIC PAGE**

I'm suggesting these edits to an existing topic:
- [x] Image (and my file is `*.png`, square, dimensions 288x288)
- [x] Content (and my changes are in `index.md`)

Please explain why these changes are necessary:

- `logo`: Was missing, I added one from [Swift's official website](https://swift.org).
- `created_by`: Other topics that come from incorporated companies do not have "Inc." in their names. Also, name on [Apple's GitHub profile](https://github.com/apple) is just "Apple".
- `related`: Objective-C is Swift's predecessor, it should be its default related topic.
- `short_description`: Old description was tightly coupling Swift with Objective-C runtime, which does not fully describe Swift's potential. New description is inspired by first sentence on [Swift's official "About" page](https://swift.org/about) and is more relevant.
- `description`: This suffered from the same as `short_description`. Again, new text is inspired by [Swift's official "About" page](https://swift.org/about).

### CODE REVIEW

File: topics/swift/index.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+related: objective-c
 display_name: Swift
-github_url: https://github.com/apple/swift
+created_by: Apple
+logo: swift.png
 released: June 2, 2014
-short_description: Swift is a dynamic programming language that allows C, C++, and
-  Objective-C to run within the same program.
-topic: swift
+short_description: Swift is a modern programming language focused on safety, performance and expressivity.
```
Comment (cheshire137) [inline topics/swift/index.md]: Would you add a `,` after "performance"? See https://github.com/github/explore/blob/master/docs/styleguide.md#commas.

File: topics/swift/index.md [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 url: https://swift.org/
+github_url: https://github.com/apple/swift
 wikipedia_url: https://en.wikipedia.org/wiki/Swift_(programming_language)
 ---
-Swift is a dynamic programming language that allows C, C++, and Objective-C to run within the same program. Its syntax takes inspiration from many of today’s more modern approaches to programming. Swift is intended to be more resilient and safer to use than Objective-C, its predecessor.
+Swift is a general-purpose programming language built using a modern approach to safety, performance and software design patterns. Inspired by many other modern programming languages, Swift is intended to be more resilient and expressive than Objective-C, its predecessor.
```
Comment (cheshire137) [inline topics/swift/index.md]: Can you add a comma after "performance"?

File: topics/swift/index.md [hunk 3]
```diff
@@ -1,13 +1,14 @@
 ---
-aliases: swift-3, swift3, swift4, swift-4, swift-language
-created_by: Apple Inc.
+topic: swift
+aliases: swift3, swift-3, swift4, swift-4, swift-language
+related: objective-c
 display_name: Swift
-github_url: https://github.com/apple/swift
+created_by: Apple
+logo: swift.png
```
Comment (cheshire137) [inline topics/swift/index.md]: Thanks for the logo!

**Comment by akashivskyy** [comment]
@cheshire137 Done!

**Comment by MikeMcQuaid** [comment]
@akashivskyy Thanks for your contribution to github/explore! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request merged**